source/_components/homekit.markdown

```diff
@@ -95,6 +102,11 @@ homekit:
                 required: false
                 type: string
                 default: ''
+              mode:
+                description: Operation mode of switches within HomeKit. Valid modes are `on_off`, `play_pause`, `play_stop`, and `toggle_mute`. The entity will ignore any `mode` which is not supported. If not specified, all supported modes will be added. Only applicable for `media_player` entities.
```

Comment:
What do you think of:
Operation **modes** of switches within **the Home app**. Valid modes are `on_off`, `play_pause`, `play_stop`, and `toggle_mute`. ~~The entity will ignore any `mode` which is not supported.~~ ~~If not specified, all supported modes will be added.~~ Only applicable for `media_player` entities.

Reply:
I agree, but I think we should stay consistent and use **HomeKit** rather than **the Home app**, because HomeKit support is not exclusive to Apple's Home app.